airflow/providers/amazon/aws/sensors/emr.py

```diff
@@ -470,3 +470,102 @@ def failure_message_from_response(response: Dict[str, Any]) -> Optional[str]:
                 f"with message {fail_details.get('Message')} and log file {fail_details.get('LogFile')}"
             )
         return None
+
+
+class EmrStepsSensor(EmrBaseSensor):
+    """
+    Asks for the state of the step until it reaches any of the target states.
+    If it fails the sensor errors, failing the task.
+    With the default target states, sensor waits step to be completed.
+    .. seealso::
+        For more information on how to use this sensor, take a look at the guide:
+        :ref:`howto/sensor:EmrStepSensor`
+    :param job_flow_id: job_flow_id which contains the step check the state of
+    :param step_id: step to check the state of
+    :param target_states: the target states, sensor waits until
+        step reaches any of these states
+    :param failed_states: the failure states, sensor fails when
+        step reaches any of these states
+    """
+
+    template_fields: Sequence[str] = ("job_flow_id", "target_states", "failed_states")
+    template_ext: Sequence[str] = ()
```

Comment:
```suggestion
```
This is the default in `BaseOperator`. No need to specify an empty `template_ext` attr.